dashboard/app/controllers/projects_controller.rb

```diff
@@ -15,4 +21,16 @@ def template
 
     render template: "projects/#{params[:template]}", layout: nil
   end
+
+  def new
```

Comment:
a couple options i can think of...

1. do it in routes
````
get '/p/playlab', to: 'LevelsController#show' , params: {id: "Playlab Standalone"}
````
(haven't checked syntax)
(and change LevelsController#show to handle "key" in addition to id in the same way we did in ScriptsController#show)

2. have a constant for eg. PLAYLAB_STANDALONE_LEVEL_NAME somewhere and just link to that (kind of like what we're doing with the rocket and pencil links now)
(also do the key/id thinkg as above)